src/components/CreateCoursePage/index.jsx

```diff
@@ -39,6 +40,15 @@ class CreateCoursePage extends React.Component {
     this.setState({ startedFetching: true });
   }
 
+  isOrgWhitelisted() {
+    const userOrgs = this.props.publisherUserInfo.organizations;
+    if (!typeWhiteList || (typeWhiteList && typeWhiteList.length === 0)) {
```

Comment:
`typeWhiteList` will always exist and be truthy, right? The definition at the top guarantees it. So you might only need the length check. And then, you don't even need the length check, right, because the `some` query will gracefully return false if the list never includes the key.